File: solutions/beecrowd/1008/1008.hs
import Text.Printf

main :: IO ()
main = do
    a <- readLn :: IO Int
    b <- readLn :: IO Double
    c <- readLn :: IO Double
    putStrLn $ id ("NUMBER = " ++ show(a))
    printf "SALARY = U$ %.2f\n" (b * c)

File: solutions/beecrowd/1008/1008.php
<?php
$a = trim(fgets(STDIN));
$b = trim(fgets(STDIN));
$c = trim(fgets(STDIN));

echo "NUMBER = " . $a . "\n";
echo "SALARY = U$ " . number_format($b * $c, 2, '.', '') . "\n";
?>
